### ISSUE TITLE
Ergänzende Metadaten

### ISSUE THREAD

**Issue by thvitt** [faustedition/xml#436]
Die document-Dateien enthalten teilweise Exzerpte aus anderen Quellen: z.B. der Diss von ABR. Müssten vor der VÖ klären, was damit passieren soll; z.b. ob die in den Quelldaten bleiben dürfen, ob sie auch im LZA- bzw. Downloadpaket sein sollen.

**Comment by gerritbruening** [comment]
Nein, sollten nicht mit ausgeliefert werden, da z.T. falsch zugeordnet usw., z.T. wohl auch nicht ganz aktuell (was Daten aus Weimar angeht).
Idealerweise einfach raus und irgendwo anders archivieren.

**Comment by thvitt** [comment]
das hab ich jetzt rauszunehmen vergessen. muss das noch vor’m release?

**Comment by gerritbruening** [comment]
Nein!
